tooling/lsp/src/requests/signature_help/tests.rs

```diff
@@ -0,0 +1,197 @@
+#[cfg(test)]
+mod signature_help_tests {
+    use crate::{
+        notifications::on_did_open_text_document, requests::on_signature_help_request, test_utils,
+    };
+
+    use lsp_types::{
+        DidOpenTextDocumentParams, ParameterLabel, Position, SignatureHelp, SignatureHelpParams,
+        TextDocumentIdentifier, TextDocumentItem, TextDocumentPositionParams,
+        WorkDoneProgressParams,
+    };
+    use tokio::test;
+
+    async fn get_signature_help(src: &str) -> SignatureHelp {
+        let (mut state, noir_text_document) = test_utils::init_lsp_server("document_symbol").await;
+
+        let (line, column) = src
+            .lines()
+            .enumerate()
+            .filter_map(|(line_index, line)| {
+                line.find(">|<").map(|char_index| (line_index, char_index))
+            })
+            .next()
```

Comment:
```suggestion
            .find_map(|(line_index, line)| {
                line.find(">|<").map(|char_index| (line_index, char_index))
            })
```
`filter_map -> next` can be `find_map`. I think there is a clippy warning for this